notes/transcript15.md

```diff
@@ -213,108 +213,228 @@ Igor: I guess the signature won't signatures won't match but we can do it a shot
+
+Ricardo: Yeah I agree with you. I just wanted to know who we should talk with to discuss this later on.
+
+Oskar: So I imagine Igor might want to do a few more runs on the post-mortem or whatever you want to call it pre-mortem, I don't know what you gonna call it, but then we can bring it up next core dev call and then we might we can also bring it up in a protocol call if that's interesting but I think.
+
+Ricardo: So basically use the Disqus post in this meeting.
+
 Igor: yeah I think Disqus is a good start and then we'll figure out from there. Because it's already a discussion started so there is no point to switch it somewhere else.
+
 Oskar: All right. Cool. So we have maybe 20 more minutes So Ricardo, you had a brief announcement on topic democracy.
+
 Ricardo: So tomorrow there is a meeting in the Status agenda that I'm going to present the swarm, what is the idea, that is being to the develop it in top democracy, basically how it works and why it works like this and so if you are interested in democracy and centralize the autonomous organizations and this can be used in many creative ways once it's created so might be interesting to many people.
+
 Oskar: Cool. Pedro you want to talk briefly about exposing cells goers in next package.
+
 Pedro: Yeah. So I think this is the the next logical step in terms of having reproducible builds. Right now we we host status code artifacts in digitalocean. So it's kind of a hand roll solution which really isn't required now that we have we hosted in this in this mix cache so we could just make status go another mix package and ideally also make the same the same approach that we took for status react does the same thing on status go so that people can just still do mix it up and mix takes care of all the dependencies and then that produces a next package which is uploaded to the next cache that we host and then that's what we use in Status react. So no more downloads during the the builds that are handled by us. It's a matter of consistency with all the other packages. I don't know if there's anything anything against this or if it's like logical for everyone. I don't know. It might be like for people who are working barely on Status Go that's maybe the people who will be more impacted by this. So it would just require that you do make shell and ensure that you're inside of this special shell from mix to build your app but it's not a big deal. You know from the experience with Status React, it's not a deal breaker. But I would be interested in hearing if there is anything against going forward with this.
+
 Igor: No just one question. Is there so when we publish a new version it just get auto-published immediately or do it, how does it work because right now a publisher just when you have for instance and it's like there is no approval process, nothing like this and if we add it to the mix as a new version,
```

Comment:
```suggestion
Igor: No just one question. Is there so when we publish a new version it just get auto-published immediately or how does it work? Because right now a publisher just when you have for instance and it's like there is no approval process, nothing like this, and if we add it to Nix as a new version...
```